src/Nancy/StaticContentConfiguration.cs

```diff
@@ -0,0 +1,25 @@
+namespace Nancy
+{
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// Safe path configuration.
+    /// </summary>
+    public class StaticContentConfiguration
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="StaticContentConfiguration"/> class.
+        /// </summary>
+        /// <param name="safePaths">StaticContent.</param>
+        public StaticContentConfiguration(IEnumerable<string> safePaths)
+        {
+            this.SafePaths = safePaths;
+        }
+
+        /// <summary>
+        /// Gets or sets the SafePaths.
```

Comment:
This is an auto-generated XML comment if I've ever seen one! :D GhostDoc has no place in our code. More descriptive and explicit comment please :D